### PR TITLE
Dummy json

### PR DESCRIPTION
**Pull request by Caleb-Cohen**
# Description

Merges @JoeDravarol and @code2244 work of creating a JSON Dummy Server into add-event-form

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Caleb-Cohen** [comment]
@InstincDev and @MatthewBozin please review to complete the merge.

File: frontend/src/services/events.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+const get = async () => {
+    const response = await axios.get('http://localhost:2121/events')
+    return response.data;
+}
+    
+// Post request
+const create = async userData => {
+    const response = await axios.post('http://localhost:2121/events', {
+        ...userData,
+        id: uuidv4()
```
Comment (JoeDravarol) [inline frontend/src/services/events.js:14]: The uuid package can be removed. It turns out json-server will generate the id for us.

File: frontend/src/services/events.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import axios from 'axios'
+import { v4 as uuidv4 } from 'uuid';
```
Comment (JoeDravarol) [inline frontend/src/services/events.js:2]: Remove uuid here.

File: frontend/package.json [hunk 3]
```diff
@@ -42,6 +44,8 @@
   "devDependencies": {
     "autoprefixer": "^10.4.13",
     "postcss": "^8.4.18",
-    "tailwindcss": "^3.2.2"
+    "tailwindcss": "^3.2.2",
+    "axios": "^1.1.3",
+    "uuid": "^9.0.0"
```
Comment (JoeDravarol) [inline frontend/package.json:49]: Remove uuid here.

**Review by JoeDravarol — CHANGES_REQUESTED** [review]

**Comment by Caleb-Cohen** [comment]
@JoeDravarol, check the commits for your requests.

File: frontend/src/services/events.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+const get = async () => {
+    const response = await axios.get('http://localhost:2121/events')
+    return response.data;
+}
+    
+// Post request
+const create = async userData => {
+    const response = await axios.post('http://localhost:2121/events', {
+        ...userData
+    })
```
Comment (JoeDravarol) [inline frontend/src/services/events.js:13]: @Caleb-Cohen That looks good. Just a minor nitpick. You can do this instead:

```suggestion
const response = await axios.post('http://localhost:2121/events', userData)
```

**Comment by InstincDev** [comment]
This looks clean and good to go.

**Pull request merged**